.github/workflows/main-branch-deployment.yml

```diff
@@ -0,0 +1,51 @@
+name: Main Branch Deployment
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - '[0-9]+-**'
```

Comment:
This should be removed from the code before merging. The workflow dispatch as well.

Reply:
It was for testing. Done.